experimental/traffic-portal/package.json

```diff
@@ -40,7 +40,7 @@
     "lint": "ng lint",
     "e2e": "npm run e2e:build && ./node_modules/.bin/nightwatch --config './nightwatch/nightwatch.conf.js'",
     "e2e:build": "./node_modules/.bin/tsc -p ./nightwatch/tsconfig.e2e.json",
-    "e2e:ci": "npm run e2e:build && ./node_modules/.bin/nightwatch --env chrome_headless --config './nightwatch/nightwatch.conf.js'",
+    "e2e:ci": "npm run e2e:build && ./node_modules/.bin/nightwatch --env chrome_headless --config './nightwatch/nightwatch.conf.js' --window-size 1920,1080",
```

Comment:
I think we should set this in the `nightwatch.conf.js` `chrome_headless` args instead.